package/yast2-packager.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Tue Mar  7 09:37:15 CET 2017 - schubi@suse.de
+
+- Product selection: Do not reinstall already installed products
+  or products which are already included in another product.
```

Comment:
Maybe I'd add a note about the real use case as this generic description does not say whether a customer need the fix or not. Something like `Fixes SLES4SAP autoupgrade with SUSE Manager repositories.`